xmake/rules/c++/modules/modules_support/clang.lua

```diff
@@ -68,37 +68,43 @@ end
 function load(target)
     -- get module and module cache flags
     local modulesflag = get_modulesflag(target)
-    local builtinmodulemapflag = get_builtinmodulemapflag(target)
     local implicitmodulesflag = get_implicitmodulesflag(target)
-    local noimplicitmodulemapsflag = get_noimplicitmodulemapsflag(target)
 
     -- add module flags
     target:add("cxxflags", modulesflag)
 
     -- add the module cache directory
     target:add("cxxflags", builtinmodulemapflag, {force = true})
     target:add("cxxflags", implicitmodulesflag, {force = true})
-    target:add("cxxflags", noimplicitmodulemapsflag, {force = true})
+    target:add("cxxflags", "-fvisibility=default", {force = true})
```

Comment:
why add it? We should not hardcode this flags, as this will affect the user's configuration of set_symbols("hidden").

Reply:
I don't know how the modules implementation in xmake affects the linkage and use of standard libraries. Buy in my Fedora it will not link against clang's standard library and cause a lot of missing symbols.